yarn.lock

```diff
@@ -6153,11 +6170,16 @@ internal-ip@^4.3.0:
     default-gateway "^4.2.0"
     ipaddr.js "^1.9.0"
 
-interpret@1.2.0, interpret@^1.1.0:
+interpret@^1.1.0:
```

Comment:
multiple versions

Reply:
Fixed, as this one should have been flattened. I forgot to run `yarn-deduplicate` sorry.